element.h

```diff
@@ -12,8 +21,10 @@ enum Element {
   ALL=Element::FIRE | Element::WATER | Element::EARTH | Element::SPIRIT | Element::LIGHTNING,
 };
 
+// contains all elements being used in the game.
```

Comment:
I think we can elaborate on this a bit:
```suggestion
// contains all elements being used in the game. Generally used to iterate through all elements.
```